### DISCUSSION TITLE
fork 仓库后执行 pnpm lint 报错

### DISCUSSION THREAD

**Discussion by daiwanxing** [jdf2e/nutui#3110] [Q&A]
![image](https://github.com/jdf2e/nutui/assets/37327614/6bf2274d-6676-42eb-979a-f5b0fe4b437b)

**Comment by daiwanxing** [comment]
这个 bug 不应该存在吧，否则其他开源贡献者在提交代码之前是怎么通过 `pnpm lint` 的？

我不确定这是否为一个 bug， `pnpm install` 后 packages 目录下的 nutui-eslint-config 和 nutui-prettier-plugin 的确没有 src 目录。

**Comment by eiinu** [comment]
项目 CI 流程（Linux）和我本地设备（MacOS）没有发现这个问题。
可能是某些代码（比如 pattern）在 Windows 环境下面不合法🤔

**Reply by daiwanxing** [reply]
> 项目 CI 流程（Linux）和我本地设备（MacOS）没有发现这个问题。 可能是某些代码（比如 pattern）在 Windows 环境下面不合法🤔

我改成 `eslint ./src ./packages/**/src/*` 后解决了此错误，这个错误的确会在 Windows 环境遇到
